modules/channel-web/src/backend/api.ts

```diff
@@ -224,7 +224,10 @@ export default async (bp: typeof sdk, db: Database) => {
   async function sendNewMessage(botId: string, userId: string, conversationId, payload, credentials: any) {
     const config = await bp.config.getModuleConfigForBot('channel-web', botId)
 
-    if (!payload.text || !_.isString(payload.text) || payload.text.length > config.maxMessageLength) {
+    if (
+      (!payload.text || !_.isString(payload.text) || payload.text.length > config.maxMessageLength) &&
+      payload.type != 'postback'
+    ) {
```

Comment:
What do you think of moving the first part of the conditions outside, like isValidTextMessage ?